mmv1/products/securitycenterv2/Organization_Source.yaml

```
@@ -0,0 +1,83 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Source'
```

The tests are written to expect `google_scc_v2_organization_source_*`, but this name produces `google_scc_v2_source_*`

This name will need to be changed to 
```suggestion
name: 'OrganizationSource'
```

---

addressed